core/app/models/spree/promotion/rules/item_quantity.rb

```diff
@@ -0,0 +1,53 @@
+# A rule to apply to an order greater than (or greater than or equal to)
+# a specific quantity of line items
+module Spree
+  class Promotion
+    module Rules
+      class ItemQuantity < PromotionRule
+        preference :quantity_min, :integer, default: 1
+        preference :operator_min, :string, default: '>'
+        preference :quantity_max, :integer, default: 1000
+        preference :operator_max, :string, default: '<'
+
+
+        OPERATORS_MIN = ['gt', 'gte']
+        OPERATORS_MAX = ['lt','lte']
+
+        def applicable?(promotable)
+          promotable.is_a?(Spree::Order)
+        end
+
+        def eligible?(order, options = {})
+          quantity = order.quantity
+
+          lower_limit_condition = quantity.send(preferred_operator_min == 'gte' ? :>= : :>, preferred_quantity_min)
+          upper_limit_condition = quantity.send(preferred_operator_max == 'lte' ? :<= : :<, preferred_quantity_max)
```

Comment:
Line is too long. [115/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.